LLama/Native/NativeApi.cs

```diff
@@ -290,6 +290,14 @@ public static void llama_log_set(NativeLogConfig.LLamaLogCallback logCallback)
         [DllImport(libraryName, CallingConvention = CallingConvention.Cdecl)]
         internal static extern void llama_kv_self_clear(SafeLLamaContextHandle ctx);
 
+        [Obsolete("Use `llama_kv_self_clear` instead")]
```

Comment:
This is exposed as `KvCacheClear` on `SafeLLamaContextHandle` now, it shouldn't be re-introduced here.

Reply:
Oh it's also an obsolete function in llama.cpp anyway!